managedvms/async-rest/src/main/webapp/WEB-INF/appengine-web.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
+	<application>async-rest-demo</application>
+	<module>async-rest-demo</module>
+	<vm>true</vm>
+	<version>1</version>
+	<public-root>/static</public-root>
+	<precompilation-enabled>false</precompilation-enabled>
+	<threadsafe>true</threadsafe>
+	<sessions-enabled>false</sessions-enabled>
+</appengine-web-app>
```

Comment:
You'll also need a Dockerfile next to index.html as this is (for now) a Java8 runtime, so a custom one (the default one is only Java7)
Dockerfile content for now: (will change soon)
```
FROM gcr.io/google_appengine/jetty9-compat:githubheadasync
ADD . /app
```